README.md

```diff
@@ -126,7 +126,11 @@ To build/test the project locally on your computer:
 4. __Run the tests__<br>
 `npm test`
 
+5. __Run the linter__<br>
+`npm run lint`
 
+6. __Run the browser coverage__<br>
+`npm run coverage:browser`
```

Comment:
How about https://github.com/APIDevTools/swagger-parser/blob/main/package.json#L47?

Reply:
Thanks for the tip, just making it `npm run coverage` shall be good enough. 
I looked at this and wonder if I am duplicating it here:
`"test": "npm run test:node && npm run test:typescript && npm run test:browser && npm run lint",`

The `npm run test:browser` versus `npm run coverage`. Both run the same tests except the latter gives a coverage stats. So, either we are good with `npm run test:browser` or we just need `npm run coverage` instead?

Reply:
yeah, we might be good with `test": "npm run coverage && npm run test:typescript && npm run lint"`, but tbh it doesn't matter. It kind of depends on whether we want to have the coverage check or not.

Reply:
Our workflows are doing the coverage checks anyway, however trying to find where will we fail if the coverage falls below say 80% for example. `npm run test` is just ensuring our unit tests pass irrespective of what the coverage values are.